modules/facade/src/main/java/com/smartbear/readyapi4j/facade/execution/RecipeExecutionFacade.java

```diff
@@ -33,7 +33,24 @@
 
     public static RecipeExecutionResult executeRecipe(TestStepBuilder... testStepBuilders) {
         TestRecipe recipe = TestRecipeBuilder.newTestRecipe(testStepBuilders).buildTestRecipe();
+        return execute(recipe);
+    }
+
+    /**
+     * Builds and executes a TestRecipe from the specified TestStepBuilders using either a local or remote
+     * executor as configured. The recipe is always executed synchronously.
```

Comment:
Good Javadoc. Do we need a method for async execution too? I suppose that won't work so well with local execution though?